### PR TITLE
Airflow UI Package upgrade

### PR DESCRIPTION
**Pull request by chethanuk**

### CODE REVIEW

**Comment by koconder** [comment]
@chethanuk-plutoflume do you have the vulnerability information?

**Comment by potiuk** [comment]
BTW. If there are serious vulnerabilities the pointers to them should not be disclosed publickly. We have the ASF security Policy and unless the vulnerabilities are public they should be announced in Private. Please DON't explain it here @chethanuk-plutoflume  if this is an undisclosed vulnerability (and in the future avoid to mention vulnerability in the PR description). This makes it easier for bad actors to exploit the vulnerabilities and we do not want to make it easier for them.

If this is a public vulnerability - as of recently we have Depebdabot enabled for those and it should be raising an automated PR to fix them.

**Review by vincentkoc — APPROVED** [review]

**Comment by koconder** [comment]
@potiuk @chethanuk-plutoflume sorry was not aware. Checks have passed on build so approved the review (although i have no write access)

File: airflow/ui/package.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "react-router-dom": "^5.2.0",
     "react-select": "^4.3.0",
     "react-table": "^7.7.0",
-    "use-react-router": "^1.0.7"
+    "use-react-router": "^1.0.7",
+    "json-schema": "^0.4.0"
```
Comment (bbovenzi) [inline airflow/ui/package.json:31]: why was `json-schema` added?

Comment (chethanuk) [reply airflow/ui/package.json:31]: Right, now json-schema is using 0.2.3 so making it 0.4.0

Is there any concept similar to constraints like in pip?

**Review by potiuk — COMMENTED** [review]
LGTM if @bbovenzi approves :)

**Review by bbovenzi — APPROVED** [review]

**Pull request merged**